tests/utilities.py

```diff
@@ -20,6 +20,10 @@ def asserter(self):
     return asserter
 
 
+def joukowsky(z):
+    return 0.5 * (z + 1 / z)
```

Comment:
recurring function in several tests, so I moved it out. Perhaps there is others that can be put here, too?